filters/block/block_test.go

```diff
@@ -39,22 +39,26 @@ func TestMatcher(t *testing.T) {
 		{
 			name:    "empty string",
 			content: "",
+			block:   []byte(".class"),
 			err:     nil,
 		},
 		{
 			name:    "small string",
 			content: ".class",
-			err:     proxy.ErrBlocked,
+			block:   []byte(".class"),
+			err:     io.ErrBlocked,
```

Comment:
This is really confusing

Reply:
What do you mean?

Reply:
I mean there are io Err* variable in stdlib https://pkg.go.dev/io#pkg-variables and this import is confusing

Reply:
ok, fixed